image/Make.defs

```diff
@@ -11,11 +11,10 @@ include $(TOPDIR)/Make.defs
 ELKS_DIR = $(TOPDIR)/elks
 ELKSCMD_DIR = $(TOPDIR)/elkscmd
 
-MINIX_BOOT = $(ELKSCMD_DIR)/bootblocks
-FD_BOOTBLOCK = $(MINIX_BOOT)/minix.bin
-FD_BOOTSECT = $(MINIX_BOOT)/fat.bin
+FD_MINIX_BOOT = $(ELKSCMD_DIR)/bootblocks/minix.bin
+FD_FAT_BOOT = $(ELKSCMD_DIR)/bootblocks/fat.bin
```

Comment:
You should keep a variable to hold the folder that contains the boot sectors / blocks, as it is going to move upward.

Reply:
I don't understand on this one. MINIX_BOOT isn't exported, nor used elsewhere? This is simpler and can be changed when needed, IMO.

Reply:
As you want...